boards/rpi_5.conf

```diff
@@ -21,6 +21,9 @@ CONFIG_FILE_SYSTEM_SHELL=y
 
 CONFIG_NOCACHE_MEMORY=y
 CONFIG_FS_MULTI_PARTITION=y
-CONFIG_DOM_STORAGE_FATFS_LOGICAL_DRIVE_NAME="0"
-CONFIG_DOM_STORAGE_FATFS_DIR="dom0/"
+CONFIG_DOM_STORAGE_FATFS_LOGICAL_DRIVE_NAME="1"
```

Comment:
This is NOT related to this PR as you switch to dom0 partition, but default ENABLE_DOM0_PART=no

Reply:
sry. Could you elaborate?